docs/docs/cmd/spo/file/file-remove.md

```diff
@@ -44,5 +50,5 @@ m365 spo file remove --webUrl https://contoso.sharepoint.com/sites/project-x --u
 Move the file with server-relative URL _/sites/project-x/SharedDocuments/Test.docx_ located in site _https://contoso.sharepoint.com/sites/project-x_ to the recycle bin
 
 ```sh
-m365 spo file remove --webUrl https://contoso.sharepoint.com/sites/project-x --url /sites/project-x/SharedDocuments/Test.docx --recycle
+m365 spo page template remove --webUrl https://contoso.sharepoint.com/sites/project-x --url /sites/project-x/SharedDocuments/Test.docx --recycle
```

Comment:
Test.Docx is not really a page template, let's add a separate sample to showcase removing a page template.